common/scripted_triggers/EAI_law_triggers.txt

```diff
@@ -33,29 +33,29 @@ EAI_can_upgrade_trade_law = {
 EAI_can_take_civilian_economy = {
 	########## AI LOGIC
 
-	always = no # dont take this ever
-
-	NOT = {
-		EAI_can_take_low_economic_mobilisation = yes
-		EAI_can_take_partial_economic_mobilisation = yes
-		EAI_can_take_war_economy = yes
-		EAI_can_take_tot_economic_mobilisation = yes
-	}
-
-	NOT = {
-		has_idea = low_economic_mobilisation
-		has_idea = partial_economic_mobilisation
-		has_idea = war_economy
-		has_idea = tot_economic_mobilisation
-	}
+	#always = no #this isnt vanilla
 
+	OR = {
+		has_idea = economy_fatigue_90
+		has_idea = economy_fatigue_91
+		has_idea = economy_fatigue_92
+		has_idea = economy_fatigue_93
+		has_idea = economy_fatigue_94
+		has_idea = economy_fatigue_95
+		has_idea = economy_fatigue_96
+		has_idea = economy_fatigue_97
+		has_idea = economy_fatigue_98
+		has_idea = economy_fatigue_99
+		has_idea = economy_fatigue_100
+	}
+	
 	NOT = { original_tag = USA }
 
 	NOT = { EAI_pp_advisor_prio = yes }
 
 	########## LAW REQUIREMENTS
 
-	NOT = { has_idea = civilian_economy }
+	#NOT = { has_idea = civilian_economy }
```

Comment:
Shouldn't this be left uncommented? We don't want this attempting to trigger a switch to civ eco if the country is already on civ eco